packages/lib/src/core/Services/http.ts

```diff
@@ -60,4 +94,7 @@ function handleFetchError(message: string, level: string) {
 }
 
 export const httpGet = (options: HttpOptions, data?): Promise<any> => http({ ...options, method: 'GET' }, data);
```

Comment:
I guess you could add the same generics `<T>` to `httpGet` as you did for `httpPost`

Reply:
Done!